test/bats.bats

```diff
@@ -221,11 +320,11 @@ fixtures bats
   [ $status -eq 0 ]
 }
 
-@test "load supports scripts on the PATH" {
+@test "load loads scripts on the BATS_LIB_PATH" {
```

Comment:
I am wary of changing the default. I don't know how many projects out there rely on loading from PATH right now. I saw your discussion for avoiding PATH but I'd like to avoid breaking existing code.

I would say we should deprecate loading from PATH and fully disable it with the next major release. Maybe we could add a switch for using PATH or BATS_LIB_PATH to ease the transition.